gulpfile.js

```diff
@@ -0,0 +1,9 @@
+const gulp = require('gulp');
+const mustache = require('gulp-mustache');
+require('dotenv').config();
+
+gulp.task('replace', function() {
+  return gulp.src(['./src/script.js', './src/index.html', './src/style.css'])
+    .pipe(mustache({ SERVER_URL: process.env.SERVER_URL }))
```

Comment:
I think you missed the point of my previous comment: 

* mustache takes an object
* process.env is an object

You can just pass the whole thing! And then anyone wanting ANY environment variable will have access to ALL of them via mustache.

Reply:
Ok, perfect I'll fix it!